tools/validators/instance_validator/tests/fake_bad_modified_ontology/FAKE_CARSON/entity_types/Carson.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
rename the Carson.yaml to ==> GOOD or BAD  + append(building_config.yaml)

Reply:
Add a comment in the header pointing out the issues in the file in case bad_building_config.yaml
or a in the header explain what the file is used for

Reply:
Done